src/app/template-library/template-library.service.ts

```diff
@@ -44,6 +43,16 @@ export class TemplateLibraryService {
     private demo: DemoService,
   ) { }
 
+  static isInCategory(templateType: string, categoryType: string): boolean {
+    return TemplateLibraryService._standardiseType(templateType) === TemplateLibraryService._standardiseType(categoryType);
+  }
+
+  private static _standardiseType(type: string): string {
```

Comment:
Since this is a function that may be re-used for different use cases, we can move it to the utilsService instead